safety_limiter/test/src/test_safety_limiter.cpp

```diff
@@ -225,6 +232,11 @@ TEST_F(SafetyLimiterTest, SafetyLimitLinear)
     EXPECT_EQ(diagnostic_msgs::DiagnosticStatus::OK, diag_->status[0].level)
         << "message: " << diag_->status[0].message;
 
+    ASSERT_TRUE(hasStatus());
+    ASSERT_TRUE(status_->is_cloud_available);
```

Comment:
It would be better to make it `EXPECT` to check all fields of `status_` even if it is partially wrong.
(other checks of `status_` fields would be same.)